pdctl/command/global.go

```diff
@@ -139,6 +141,26 @@ func validPDAddr(pd string) error {
 	return nil
 }
 
+func postJSON(cmd *cobra.Command, prefix string, input map[string]interface{}) {
+	data, err := json.Marshal(input)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	url := getAddressFromCmd(cmd, prefix)
+	r, err := http.Post(url, "application/json", bytes.NewBuffer(data))
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+	defer r.Body.Close()
+
```

Comment:
should read all r.Body, I remember if not, the connection may be leak even close the body.